Automattic/woocommerce-payments
# Rename deposit to payout on the deposits list page.

Fixes https://github.com/Automattic/woocommerce-payments/issues/9524 and https://github.com/Automattic/woocommerce-payments/issues/9577.

#### Changes proposed in this Pull Request

Rename all 'deposit' to 'payout' on deposits list page.

Page title:

![Screenshot 2024-10-16 at 02 01 54](https://github.com/user-attachments/assets/67b0c446-0bdb-4699-a243-d8dd611dace7)


Page body:

The page body [renders 4 components](https://github.com/Automattic/woocommerce-payments/blob/2087e224725d8c3497d01e8b230cb39faca088c7/client/deposits/index.tsx#L151-L154) some of which might be hidden in certain condition:
- `<TestModeNotice/>`
- `<NextDepositNotice/>`
- `<DepositFailureNotice/>`
- `<DepositsList/>`

<img width="1496" alt="Screenshot 2024-10-10 at 22 18 43" src="https://github.com/user-attachments/assets/f9d0a8cb-a9d4-4068-b9a3-d89d272a78c7">

This what `<NextDepositNotice/>` renders (when tooltip is opened):

![376755763-7a512d63-9cff-41b1-b526-840d7c3adb3a](https://github.com/user-attachments/assets/1b9dc6b5-c96a-44ca-ba6e-3c249905c959)

This what `<DepositFailureNotice/>` renders, when there's deposit failure:

![Screenshot 2024-10-16 at 02 16 36](https://github.com/user-attachments/assets/b213f8c2-1cc3-4572-b910-df6644f39cb5)


Filter UI:

<img width="488" alt="Screenshot 2024-10-10 at 22 19 44" src="https://github.com/user-attachments/assets/35ce247e-eae9-4193-b783-e86f53be42d5">

Deposit schedule component's tooltip content (I created https://github.com/Automattic/woocommerce-payments/issues/9559 for the links color that is too bright):

![Screenshot 2024-10-16 at 02 03 03](https://github.com/user-attachments/assets/ff9561cc-ee83-4d14-afc2-0d43c12738d3)


#### Testing instructions

* Open deposits list page (wp-admin > Payments > Deposits).
* Make sure there is no 'deposit' mentioned on that page as they are renamed to 'payout'.
* Try to open filter UI as well.
* Click on the deposit schedule tooltip and make sure all 'deposit's are renamed to 'payout'.
* Either make a deposit failed or make a local change to these [lines](https://github.com/Automattic/woocommerce-payments/blob/2087e224725d8c3497d01e8b230cb39faca088c7/client/deposits/index.tsx#L112-L142) so `<DepositFailureNotice/>` renders. Make sure all 'deposit's are renamed.

-------------------


**Post merge**
